test/unit/adapters/REST/endpoints/concept-scheme-test.ts

```diff
@@ -0,0 +1,153 @@
+import { describe, test } from 'mocha'
+import { cloneMock } from '../../../mocks/entities'
+import { expect } from 'chai'
+import setupRestAdapter from '../helpers/setupRestAdapter'
+
+function setup<T>(promise: Promise<T>, params = {}, type = 'conceptScheme') {
+  return {
+    ...setupRestAdapter(promise, params),
+    entityMock: cloneMock(type),
+  }
+}
+
+describe('ConceptScheme', () => {
+  test('get', async () => {
+    const { httpMock, adapterMock, entityMock } = setup(Promise.resolve({}))
+    httpMock.get.returns(Promise.resolve({ data: entityMock }))
+
+    return adapterMock
+      .makeRequest({
+        entityType: 'ConceptScheme',
+        action: 'get',
+        params: {
+          organizationId: 'organization-id',
+          conceptSchemeId: 'concept-scheme-id',
+        },
+      })
+      .then((r) => {
+        expect(r).to.eql(entityMock)
```

Comment:
not sure what actually this tests

Reply:
i would maybe prefer to see that url and query params are resolved correctly. 

also, since it's a public lib in TS, maybe it's worth testing the TS interface